source/features/fit-textareas.tsx

```diff
@@ -22,5 +17,8 @@ function init(): void {
 
 features.add({
 	id: 'fit-textareas',
+	description: `Automatically expands textareas to fit ALL of their content, rather than have a height limit like GitHub's native "fit to content" behavior.
```

Comment:
```suggestion
	description: `Automatically expand textareas to fit all of their content, rather than have a height limit like GitHub's native "fit to content" behavior.
```